### PR TITLE
Adds bottom nav border in high contrast mode

### LINKED ISSUE #4291: Bug: Navigation Banner transparent bottom border missing when user changes colours
**Describe the bug**
When I view the site in Firefox with colour preferences changed and with Windows High Contrast mode I should be able to see where the banner ends, this should be represented with a line, like the start of the footer (see screen shot). This should be the same on all page types of the site. Note some page types appear to have a 2px border at the top of the navigation in the banner (see screen shot), this should only be 1px, this should also be addressed.

Note: Background colours aren’t implemented when you change colours and therefore you won’t see any background colours, hence the need for transparent border - this picks up the colours you choose

**Screenshots**
Missing border at bottom of banner
Screenshot 2021-01-26 at 11 41 44

Footer example
Screenshot 2021-01-26 at 11 41 39

2px border at the top of the navigation on some pages - this should be 1px
Screenshot 2021-01-26 at 11 51 45

**Testing notes**
- [ ] View the banner with the colour preferences changed in FireFox. Go to the ‘Preferences’ under the ‘Firefox’ menu, then under ‘Language and Appearance’, select ‘Colours’ and change the colours for text, background, unvisited and visited links, then select ‘Always’ for ‘Override the colours specified by the page with your selections above’. After testing, you will need to restore the default colors.
- [ ] View the banner with High Contrast mode on Windows. Go to the ‘Personalization’ settings in the Control Panel and view with custom colours.

### PR DESCRIPTION
**Pull request by mukama**
Resolves https://github.com/bbc/psammead/issues/4291

**Overall change:** 
This PR adds a bottom border to the navigation component. This is visible only in high contract mode.

**Before**
image

**After**
image

**Code changes:**

- Adds an `after` pseudo-element of with no height and a transparent background. This is only visible in high contrast mode.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by HarryVerhoef — APPROVED** [review]
Nice one! Think this just needs the psammead-navigation package to be bumped then the CI will pass 👍

File: package.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
     "@bbc/psammead-media-indicator": "6.1.0",
     "@bbc/psammead-media-player": "5.1.0",
     "@bbc/psammead-most-read": "6.0.28",
-    "@bbc/psammead-navigation": "9.1.1",
+    "@bbc/psammead-navigation": "9.1.2",
```
Comment (andrewscfc) [inline package.json]:
```suggestion
    "@bbc/psammead-navigation": "9.2.0",
```
I think this should be a minor package bump as component is changing behaviour (not just an internal bug fix) see https://semver.org/

File: packages/components/psammead-navigation/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 9.1.2 | [PR#4475](https://github.com/bbc/psammead/pull/4475) Adds bottom nav border in high contrast mode |
```
Comment (andrewscfc) [inline packages/components/psammead-navigation/CHANGELOG.md]:
```suggestion
| 9.2.0 | [PR#4475](https://github.com/bbc/psammead/pull/4475) Adds bottom nav border in high contrast mode |
```

File: packages/components/psammead-navigation/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-navigation",
-  "version": "9.1.1",
+  "version": "9.1.2",
```
Comment (andrewscfc) [inline packages/components/psammead-navigation/package.json]:
```suggestion
  "version": "9.2.0",
```

**Review by andrewscfc — COMMENTED** [review]
Looks good, just a suggestion on the version number

**Review by andrewscfc — APPROVED** [review]
Thanks @mukama

File: package.json (hunk 1 above)
Comment (mukama) [reply package.json]: Great suggestion @andrewscfc. Thanks for the review

**Comment by greenc05** [comment]
Great, thanks @mukama Please could you confirm if this will be implemented across all page types? I will just check this is as expected in Edge on Windows in High Contrast Mode... I assume those screen shots are from Firefox with the colour preferences changed?

**Comment by greenc05** [comment]
I can confirm this looks as expected on the preview url in Edge and IE11 on Windows 10 in a custom high contrast mode.

**Comment by mukama** [comment]
> Great, thanks @mukama Please could you confirm if this will be implemented across all page types? I will just check this is as expected in Edge on Windows in High Contrast Mode... I assume those screen shots are from Firefox with the colour preferences changed?

That is correct! The screenshots are from Firefox with custom colour preferences turned on

**Pull request merged**